packages/story-editor/src/components/library/panes/shopping/productList.js

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useEffect } from '@googleforcreators/react';
+import PropTypes from 'prop-types';
+import {
+  Text,
+  THEME_CONSTANTS,
+  Icons,
+  Button,
+  BUTTON_TYPES,
+  BUTTON_SIZES,
+  BUTTON_VARIANTS,
+} from '@googleforcreators/design-system';
+import styled, { css } from 'styled-components';
+import { __, sprintf } from '@googleforcreators/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useProductNavigation from './useProductNavigation';
+import { formatPrice, isOnPage } from './util';
+
+const StyledListItem = styled.div`
+  display: flex;
+  width: 100%;
+  padding: 5px 0;
+  align-items: center;
+  border-bottom: 2px solid ${({ theme }) => theme.colors.divider.tertiary};
+  transition: background-color ease-in-out 300ms;
+  &:focus-within,
+  &:hover,
+  &:focus {
+    background-color: ${({ theme }) => theme.colors.interactiveBg.disable};
+  }
+`;
+
+const baseIcon = css`
+  width: 16px;
+  height: 16px;
+  border-radius: 100%;
+  color: ${({ theme }) => theme.colors.interactiveBg.disable};
+`;
+
+const StyledActionsContainer = styled.div`
+  margin-right: 10px;
+
+  .check {
+    ${baseIcon}
+    background-color: ${({ theme }) => theme.colors.fg.positive};
+  }
+
+  .remove {
+    ${baseIcon}
+    background-color: ${({ theme }) => theme.colors.fg.negative};
+    display: none;
+  }
+
+  button {
+    margin-right: 10px;
+    &:focus,
+    &:hover {
+      .remove {
+        display: block;
+      }
+      .check {
+        display: none;
+      }
+    }
+  }
+`;
+
+const imgPlaceholder = css`
+  display: block;
+  margin-right: 10px;
+  border-radius: 4px;
+  background-color: ${({ theme }) => theme.colors.interactiveBg.disable};
+  width: 41px;
+  height: 41px;
+`;
+
+const StyledImgPlaceHolder = styled.div`
+  ${imgPlaceholder}
+`;
+
+const StyledImage = styled.img`
+  ${imgPlaceholder}
+`;
+
+const StyledDescription = styled.div`
+  padding-left: 12px;
+`;
+
+const StyledTitle = styled(Text).attrs({
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.MEDIUM,
```

Comment:
According to Figma the font size is suppose to be 14px, but `MEDIUM` is 16px.

```suggestion
  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL,
```